### PR TITLE
Eliminate ChangeMultipleNodesOptions in favor of smart separators

### PR DESCRIPTION
**Pull request by amcasey**
Every caller uses newline separators and no caller wants to add one if there's already a newline at the end of the preceding node, so just make that the default behavior.

Bonus: Change the default options for the plural versions to match those for the singular versions.

### CODE REVIEW

**Comment by amcasey** [comment]
The primary beneficiary of this change is the organize imports command I'm working on now.

**Comment by amcasey** [comment]
The second commit specifically makes `useNonAdjustedPositions` *not* the default so that the default matches the singular APIs.  If you don't think that consistency is valuable, we can just drop the second commit.

**Comment by andy-ms** [comment]
My opinion is that as much as possible, we should not require the users to pass options -- because when you're writing a codefix you probably aren't thinking about that, so it's better to just try and make the method smarter about recognizing the kind of situation it's in, like `getOptionsForInsertNodeBefore` does.

**Comment by amcasey** [comment]
I'll by that.  How about, as a compromise, I make it the default but tag the singular APIs with #21246 and say that they should probably use the same default?

**Comment by andy-ms** [comment]
Sure.

File: src/services/textChanges.ts [hunk 1]
```diff
@@ -72,6 +72,11 @@ namespace ts.textChanges {
      */
     export type ConfigurableStartEnd = ConfigurableStart & ConfigurableEnd;
 
+    export const useNonAdjustedPositions: ConfigurableStartEnd = {
```
Comment (andy-ms) [inline src/services/textChanges.ts]: No longer needs to be exported

Comment (amcasey) [reply src/services/textChanges.ts]: I left it exported because I'll be consuming it in another PR.

**Pull request merged**